lib/mapper.js

```diff
@@ -107,21 +109,43 @@ exports.track = function(track) {
 /**
  * Map group `msg`.
  *
+ * Intercom request we still use bulk API for even groups for scalability reasons
+ * https://developers.intercom.com/reference#bulk-user-operations
+ *
  * @param {Group} msg
  * @return {Object}
  */
 
 exports.group = function(msg) {
-  var ret = {};
-  ret.remote_created_at = time(msg.created());
-  ret.company_id = msg.groupId();
-  ret.name = msg.name();
-  ret.monthly_spend = msg.proxy('traits.monthlySpend');
-  ret.plan = msg.proxy('traits.plan');
-  ret.custom_attributes = formatTraits(msg.traits());
-  delete ret.custom_attributes.companies;
-  delete ret.custom_attributes.company;
-  ret.custom_attributes = flatten(ret.custom_attributes);
+  var ret = { items: [] };
+  var customTraits = flatten(formatTraits(msg.traits()));
+
+  // Remove duplicate/semantic traits
+  var semanticTraits = ['companies', 'company', 'remote_created_at', 'createdAt', 'created_at', 'company_id', 'name', 'monthly_spend', 'monthlySpend', 'plan', 'custom_attributes', 'id'];
```

Comment:
Not super important, but I find long lines hard to read. Long arrays look nice as
```
var array = [
  'elem1',
  'elem2',
  ...
];
```